jenkins-integration/jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -x
+
+while true ; do
+  curl -sS -w "\n" -k https://localhost:8140/status/v1/services?level=debug >> /var/log/puppetlabs/puppetserver/metrics.json
```

Comment:
I've thought about this a little and my current opinion is that we should wait until we start trying to write tooling around this, and see what kinds of issues we actually encounter, before changing it.  I've used this exact curl command for a bunch of debugging in the past (e.g. graphing memory usage when we landed our HUP work) and haven't had any issues so far.